daprdocs/content/en/reference/components-reference/supported-state-stores/setup-redis.md

```diff
@@ -244,11 +244,14 @@ Consider the example from ["How-To: Query state"]({{< ref "howto-state-query-api
 {{< tabs "Self-Hosted" "Kubernetes" "Azure" "AWS" "GCP" "Redis Enterprise Cloud" "Alibaba Cloud" >}}
 
 {{% codetab %}}
-If you are using a self-hosted deployment of Dapr v1.7 and up, a Redis instance with the required modules is automatically created as a Docker container when you run `dapr init`.
+The Redis instance that gets created automatically with `dapr init` cannot be used with query API. You can run `redislabs/rejson` docker image on a different port(than already installed `redis` is using) to work with query API. 
```

Comment:
```suggestion
If you are using a self-hosted deployment of Dapr v1.7 and up, a Redis instance without the JSON module is automatically created as a Docker container when you run `dapr init`.

Alternatively, you can create an instance of Redis by running the following command:
 ```bash
 docker run -p 6379:6379 --name redis --rm redis
 ```

The Redis container that gets created on `dapr init` or via the above command, cannot be used with state store query API alone. You can run `redislabs/rejson` docker image on a different port(than already installed `Redis` is using) to work with query API. 
```

Reply:
I would remove the mention to v1.7. This applies to all versions of Dapr and just makes it look stale one day.